client/package-lock.json

```diff
@@ -1,8 +1,5143 @@
 {
   "name": "client",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
`client/package-lock.json` and `server/package-lock.json` are being completely modified, when that shouldn't happen.

You can easily undone the changes of those two files with [git-restore](https://git-scm.com/docs/git-restore). 

It happened in #9, if you have problems returning the files to their previous state, check how it was resolved.